lib/AST/DiagnosticConsumer.cpp

```diff
@@ -118,6 +118,12 @@ FileSpecificDiagnosticConsumer::subconsumerForLocation(SourceManager &SM,
   if (loc.isInvalid())
     return None;
 
+  // What if a there's a FileSpecificDiagnosticConsumer but there are no
+  // subconsumers in it? (This situation obtains for the fix-its
```

Comment:
Typo: "obtains" instead of "occurs".

Reply:
I meant "obtains", but "occurs" is better. Fixed, tx.